sites/platform/src/guides/ibexa/deploy.md

```diff
@@ -5,159 +5,8 @@ weight: -10
-
-```bash
-~/ez platformsh:setup
-```
-
-eZ Launchpad generates the files for you and you are then totally free to fine tune them.
-
-#### Solr specificity
-
-Solr is fully functional with eZ Launchpad but it isn't enabled by default on {{% vendor/name %}}.
-You have to set it up manually following the [current documentation](https://github.com/ezsystems/ezplatform/blob/master/.platform/services.yaml#L37).
-
-Actions needed are:
-
-* Generate the Solr configuration thanks to the script provided by Ibexa.
-* Put the result in the `{{< vendor/configdir >}}` at the root of your project.
-* Add the service in the `{{< vendor/configfile "services" >}}`.
-* Add the relationship in the `{{< vendor/configfile "app" >}}`.
-
-#### Environment variables (optional)
-
-eZ Launchpad allows you to define environment variables in the `provisioning/dev/docker-compose.yml` file.
-You may use that to set [variables](../../development/variables/_index.md) to match {{% vendor/name %}} environments so that you can keep your environment behavior in sync.
-
-Such variables have to be set in the `engine` container.
-
-```yaml
-# provisioning/dev/docker-compose.yml
-engine:
-    environment:
-        - ASIMPLEVARIABLE=avalue
-        - PLATFORM_RELATIONSHIPS=A_BASE64_ENCODED_VALUE
-```
-
-### Local development with {{% vendor/name %}}
-
-Thanks to eZ Launchpad, you can work 100% locally: [untethered](../../development/local/untethered.md).
-You have the whole project working offline on machine.
-
-{{< note >}}
-
-{{% vendor/name %}} also provides smooth [tethered SSH tunnels](../../development/local/tethered.md).
-
-{{< /note >}}
-
-Local services are provided by the Docker stack but there are minimum day-to-day tasks that you might need with {{% vendor/name %}}.
-
-The main ones are:
-
-* **Downstream database synchronization**: Getting it from the remote to the local.
-* **Downstream file storage synchronization**: Getting it from the remote to the local.
-
-To help you with that, {{% vendor/name %}} provides a CLI that you can [install](../../administration/cli/_index.md).
-
-#### Database and storage synchronization
-
-```bash
-{{% vendor/cli %}} db:dump --gzip -f ezplatform.sql.gz -d data/ -y
-{{% vendor/cli %}} mount:download -m ezplatform/web/var --target=ezplatform/web/var/ -y
-~/ez/importdata
-```
-
-The two first lines get the remote database and storage from the remote environment and stores it locally in `data/`.
-The third tells to eZ Launchpad to import those data in the Docker stack.
-
-{{< note >}}
-
-The storage (images and files) synchronization is optional.
-Ibexa DXP provides a [placeholder generator mechanism](https://doc.ibexa.co/en/latest/content_management/images/images/#generating-placeholder-images)
-that allows you to forget about the real images for your local.
-
-{{< /note >}}
+See how to [install Ibexa DXP on Platform.sh through Ibexa Cloud](https://doc.ibexa.co/en/latest/getting_started/install_on_ibexa_cloud/).
```

Comment:
```suggestion
See how to [install Ibexa DXP on Platform.sh through Ibexa Cloud](https://doc.ibexa.co/en/latest/getting_started/install_on_ibexa_cloud/).

## Redis

See [How to enable addtional services](https://doc.ibexa.co/en/latest/getting_started/install_on_ibexa_cloud/#additional-services)

## Solr

See [How to enable addtional services](https://doc.ibexa.co/en/latest/getting_started/install_on_ibexa_cloud/#additional-services)
```

https://doc.ibexa.co/en/latest/getting_started/install_on_ibexa_cloud/#additional-services is a bit poor. It could be more detailed.